tests/unit/addon-tree-test-node.js

```diff
@@ -19,6 +19,8 @@ function findMirage(app) {
 }
 
 describe('Addon', function() {
+  this.timeout(15000);
```

Comment:
These tests run significantly slower than they used to. Ember-CLI 0.2.7 seems to be much much slower than 0.2.0.